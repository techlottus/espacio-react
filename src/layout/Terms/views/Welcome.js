import React from "react";
import "./Welcome.scss";

const Welcome = ({title, image}) => {
  return (
    <>
      <div className="welcome-container">
        <div className="title-container"> 
          <h2 className="welcome-title">
            {title}
          </h2>
        </div>
        <div className="img-container">
            <img className='img-notebook' src={image} alt="logo-notebook" />
        </div>
      </div>
    </>
  );
};

export default Welcome;
