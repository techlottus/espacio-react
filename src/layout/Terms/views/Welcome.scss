@import '../../../App.scss';

.welcome-container{
    padding: pixeltorem(70px) pixeltorem(5px) 0 pixeltorem(5px);
    margin: 0 pixeltorem(15px);
    font-family: "Nunito", sans-serif;

    .title-container{
        .welcome-title{
            color: map-get($palette-neutral, n40);
            text-align: center;
            font-size: pixeltorem(24px);
            padding-top: pixeltorem(100px);
        }
    }
    .img-container{
        text-align: center;
        .img-notebook{
            width: 65%;
            object-fit: fill;
            object-position: 0 pixeltorem(90px);
        }
    }

}