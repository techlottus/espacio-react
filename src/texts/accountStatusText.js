export const accountStatusText = {
  statusTitle: "Estado de cuenta y consultas de facturas",
  resumeTitle: "Resumen de adeudo",
  total: "Total",
  errorMessage:
    "Tuvimos dificultades para cargar tu información en este momento",
  emptyMessage: {
    congrats: "Felicidades, estás al día con tus pagos",
    continue: "Continúa transformando tu futuro",
  },
  images: { 
    user: "images/user.png",
    empty: "yeah.png",
  },
  btnModal: {
    delete: "Eliminar",
    cancel: "Cancelar",
  },
  btnDownload: {
    download: "Descargar Estado de cuenta",
  },
  btnInvoice: {
    checkInvoice: "Consulta tus facturas",
  },
  tableAccountStatus: [
    { title: "Clave" },
    { title: "Ciclo" },
    { title: "Periodo" },
    { title: "Adeudo" },
  ],

  periodDetailTitle: "Detalles por periodo",
  periodDetailDescription:
    "Selecciona el programa y el periodo para ver los detalles de los movimientos; se clasifican de la siguiente manera:",
  periodDetailBullets: {
    debt: "Con adeudo",
    payed: "Pagados",
    toPay: "Por pagar",
  },
  periodDetailSelects: {
    program: "Programa",
    period: "Periodo",
  },
  accountDetailPeriod: {
    title: "Periodo",
    icon: "calendar_today",
    debt: "Saldo del periodo",
  },
  accountDetailMovements: {
    title: "Movimientos",
  },
  tableInvoiceQuery: [
    { title: "Fecha de emisión" },
    { title: "Periodo" },
    { title: "Concepto" },
    { title: "Monto" },
    { title: "RFC" },
    { title: "Preview" },
    { title: "PDF" },
    { title: "XML" },
  ],
  tableRowInvoiceQuery: [
    { title: "07 Abril 2021" },
    { title: "Periodo 5" },
    { title: "Colegiatura" },
    { title: "$2405.50" },
    { title: "LRKO485767HF2" },
  ],
  accountStatusInvoiceQuery: {
    title: "Consulta de facturas",
    description:
      "Filtra por rango de fechas, selecciona el perido que deseas consultar y presiona descargar.",
  },
  btnInvoiceQuery: {
    download: "Descargar todo",
    clear: "Limpiar",
  },
  linkInvoiceQuery: "Ver más"
};
