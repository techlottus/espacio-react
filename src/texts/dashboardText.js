export const dashboardText = {
  menu: {
    aula: {
      emoji: "mortar-board",
      text: "Aula",
    },
    book: {
      emoji: "books",
      text: "Biblioteca",
    },
    helpCenter: {
      emoji: "thought-balloon",
      text: "Centro de Ayuda",
    },
    procedures: {
      emoji: "file-folder",
      text: "Trámites",
    },
    accountStatus: {
      emoji: "chart-with-upwards-trend",
      text: "Estado de Cuenta",
    },
    payment:  {
      emoji: "credit-card",
      text: "Pagos",
    },
    invoice: {
      emoji: "page-facing-up",
      text: "Facturas"
    },
    benefits: {
      emoji: "star",
      text: "Portal de Beneficios"
    },
    sede: {
      emoji: "notebook",
      text: "SeDe"
    },
    psychopedagogical: {
      emoji: "blue-heart",
      text: "Centro de Atención Psicopedagógica"
    },
    evaluateServices: {
      emoji: "ec ec-white-check-mark",
      text: "Evalúa los servicios"
    },
    registrationSubjects: {
      emoji: "ec ec-closed-book",
      text: "Inscribe tus materias"
    },
    logout: {
      emoji: "back",
      text: "Cerrar sesión",
    },
  },
  titleWelcome: "Hola",
  images: {
    cardHeader: "images/header.png",
    empty: "empty.png",
  },
  logo: {
    urlLogo: "logo.svg"
  },
  logos: "logo.svg",
  sectionDashboard: {
    titleNews: "Noticias de tu universidad",
    titleCards: "¿Qué quieres hacer hoy?",
  },
  playIcon: "play_circle_outline",
  cards: {
    one: {
      title: "Aula",
      icon: "school",
    },
    two: {
      title: "Pagos",
      icon: "credit_card",
    },
    three: {
      title: "Biblioteca",
      icon: "import_contacts",
    },
    four: {
      title: "Trámites",
      icon: "post_add",
    },
    five: {
      title: "Inscribe tus materias",
      icon: "add_circle_outline",
    },
  },
  footerText: {
    text: "©2022 Derechos reservados Lottus Education",
    privacy: "Aviso de privacidad y términos y condiciones",
  },
};