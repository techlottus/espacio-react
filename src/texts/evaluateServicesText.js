export const evaluateServicesText = {
    evaluateServicesTitle: "Evalua tus servicios digitales",
    modalSurvey: {
        title: "Evalua tu experiencia para realizar tu pago a través de Espacio UTC",
        text: "¿Que tan probable es que recomiendes los pagos en Espacio UTC?"
    }
}