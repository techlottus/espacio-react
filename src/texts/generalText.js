export const generalText = {
  modals: {
    modalExit: {
      icon: "help",
      modalTitlePayment: "¿Seguro que deseas salir de la sección de pagos?",
      modalTitleProcedure: "¿Seguro que deseas abandonar el trámite?",
      exitBtn: "Aceptar",
      cancelBtn: "Cancelar",
    },
  },
};
