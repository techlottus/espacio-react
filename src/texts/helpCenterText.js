export const helpCenterText = {
  titleDashboard: "¿Cómo te podemos ayudar?",
  images: {
    headerHelpCenter: "images/back-texture-white.png",
    helpCenterFlowBanner: "images/back-texture-white.png",
    helpCenterFlowAcademic: "images/banner-ayuda-academica.png",
    helpCenterFlowFinancial: "images/banner-ayuda-financiera.png",
    helpCenterFlowInstalations: "images/banner-ayuda-instalaciones.png",
    helpCenterFlowPsychopedagogical: "images/banner-ayuda-psicometrica.png",
    helpCenterFlowTechnical: "images/banner-ayuda-tecnica.png",
    helpCenterFlowProcedures: "images/banner-ayuda-tramites.png",
    helpCenterFaqs: "images/banner-preguntas-frecuentes.png",
  },
  faqsAccordion: [
    {
      title: "¿Cuándo inician mis clases?",
      description:
        "Deberás escanear completo, a color y por ambos lados los siguientes documentos:",
      list: [
        "Partida o acta de nacimiento apostillada.",
        "Certificado o notas de bachillerato apostillado, donde deben señalar los grados y ciclos escolares cursados.",
        "Diploma o título del nivel bachillerato.",
        "Notas o analítico de secundaria o educación básica",
        "Diploma o título del nivel secundaria",
        "Identificación oficial (Si el alumno es menor de edad, deberá presentar identificación oficial del padre o tutor.",
      ],
    },
    {
      title: "¿Cuál navegador es el indicado para acceder la aula virtual ? ",
      description:
        "Deberás escanear completo, a color y por ambos lados los siguientes documentos",
      list: [
        "Acta de nacimiento.",
        "Certificado parcial del nivel superior, donde deben señalar los grados y ciclos escolares cursados.",
        "Certificado total de estudios de nivel anterior inmediato.",
        "Título de nivel anterior inmediato.",
        "Identificación oficial.",
        "Carta poder.",
      ],
    },
    {
      title:
        "¿Hay una fecha límite para entregar mis actividades?, ¿Dónde puedo consultarla? ",
      description:
        "Deberás escanear completo, a color y por ambos lados los siguientes documentos",
      list: [
        "Acta de nacimiento.",
        "Certificado parcial del nivel superior, donde deben señalar los grados y ciclos escolares cursados.",
        "Certificado total de estudios de nivel anterior inmediato.",
        "Título de nivel anterior inmediato.",
        "Identificación oficial.",
        "Carta poder.",
      ],
    },
    {
      title: "Tuve un problema con un profesor ¿Qué puedo hacer?",
      description:
        "Deberás escanear completo, a color y por ambos lados los siguientes documentos",
      list: [
        "Acta de nacimiento.",
        "Certificado parcial del nivel superior, donde deben señalar los grados y ciclos escolares cursados.",
        "Certificado total de estudios de nivel anterior inmediato.",
        "Título de nivel anterior inmediato.",
        "Identificación oficial.",
        "Carta poder.",
      ],
    },
    {
      title: "¿Dónde puedo consultar mis calificaciones y fechas de entrega?",
      description:
        "Deberás escanear completo, a color y por ambos lados los siguientes documentos",
      list: [
        "Acta de nacimiento.",
        "Certificado parcial del nivel superior, donde deben señalar los grados y ciclos escolares cursados.",
        "Certificado total de estudios de nivel anterior inmediato.",
        "Título de nivel anterior inmediato.",
        "Identificación oficial.",
        "Carta poder.",
      ],
    },
  ],
  titleFlowFaqs: "Preguntas frecuentes",
  titleFlowTickets: "Levanta un ticket",
  arrowFormHelpCenter: {
    icon: "arrow_back",
    text: "Atrás",
  },
  modalConfirmHelpCenter: {
    success: {
      icon: "check_circle_outline",
      title: "El ticket se generó correctamente",
      firstDescription: "Tu solicitud ha sido recibida y se ha creado el ticket",
      secondDescription: "Debes conservar el No. de ticket para poder consultar el seguimiento del mismo.",
      btns: {
        next: "Ir a centro de ayuda",
      },
    },
    error: {
      img: "empty-error.png",
      title: "Ha ocurrido un error, inténtalo más tarde",
    },
  },
  modalHelpCenter: {
    title: "Verifica que la información esté correcta",
    issue: "Asunto: ",
    assignment: "Materia: ",
    professor: "Profesor: ",
    ades: "Asesor: ",
    description: "Descripción del caso: ",
    activity: "Actividad: ",
    assignmentToReport: "Material a reportar: ",
    paymentDate: 'Fecha de pago: ',
    paymentMount: 'Monto de pago: ',
    paymentMethod: 'Medio de pago: ',
    classDate: 'Fecha de la clase: '
  },
  searchResults: {
    empty: "No se han encontrado resultados para tu búsqueda",
    error: "Ha ocurrido un error, inténtalo más tarde",
  },
  directory: {
    title: "Directorio UTC",
    tableTitleOne: "Gerentes y coordinadores",
    tableTitleTwo: "Asesores educativos",
    headerTableCoordinator: [
      { title: "Nombre" },
      { title: "Área" },
      { title: "Teléfono" },
      { title: "Correo" },
    ],
    headerTableAdvisor: [
      { title: "Nombre" },
      { title: "Área" },
      { title: "Contacto" },
    ],
  },
};
