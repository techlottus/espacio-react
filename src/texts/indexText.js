import { termsText } from "./termsText";
import { paymentText } from "./paymentText";
import { dashboardText } from "./dashboardText";
import { accountStatusText } from "./accountStatusText";
import { loginText } from "./loginText";
import { screenText } from "./screenText";
import { sidebarText } from "./sidebarText";
import { proceduresText } from "./proceduresText";
import { generalText } from "./generalText";
import { helpCenterText } from "./helpCenterText";
import { evaluateServicesText } from "./evaluateServicesText";
import { registrationSubjectsText } from "./registrationSubjectsText";

export const texts = {
  terms: termsText,
  payment: paymentText,
  dashboard: dashboardText,
  accountStatus: accountStatusText,
  login: loginText,
  screen: screenText,
  sidebar: sidebarText,
  procedures: proceduresText,
  general: generalText,
  helpCenter: helpCenterText,
  evaluateServices: evaluateServicesText,
  registrationSubjects: registrationSubjectsText
}