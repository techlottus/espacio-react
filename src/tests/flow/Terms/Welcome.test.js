import React from "react";
import { mount } from "enzyme";
import { Provider } from "react-redux";
import "@testing-library/jest-dom";
import configureStore from "redux-mock-store";
import thunk from "redux-thunk";
import Welcome from "../../../layout/Terms/views/Welcome";
import { termsText } from "../../../texts/termsText";

const middlewares = [thunk];
const mockStore = configureStore(middlewares);
window.dataLayer = {
  push: jest.fn()
}

const initState = {
  texts: {
    terms: {
      ...termsText
    }
  }
};
let store = mockStore(initState);


const wrapper = mount(
  <Provider store={store}>
    <Welcome />
  </Provider>
);

describe("Test <Welcome/> ", () => {
  beforeEach(() => {
    store = mockStore(initState);
  });

  test("should init component", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
