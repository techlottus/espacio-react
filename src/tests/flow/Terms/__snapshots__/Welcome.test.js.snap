// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test <Welcome/>  should init component 1`] = `
<div
  className="welcome-container"
>
  <div
    className="title-container"
  >
    <h2
      className="welcome-title"
    />
  </div>
  <div
    className="img-container"
  >
    <img
      alt="logo-notebook"
      className="img-notebook"
      src="images/Terms/terms.png"
    />
  </div>
</div>
`;
